comp/trace/ssi/component.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Please use the new layout describe in the doc https://datadoghq.dev/datadog-agent/components/creating-components/